Finjector.Web/ClientApp/src/components/Details/PpmDetails.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { renderNameAndEmail } from "../../util/util";
+import { PpmDetails } from "../../types";
+import CopyToClipboardHover from "../../shared/CopyToClipboardHover";
+
+interface PpmDetailsProps {
+  details: PpmDetails;
+}
+
+const PpmDetailsPage: React.FC<PpmDetailsProps> = ({ details }) => {
+  return (
+    <>
+      <div className="row">
+        <div className="col-3 coa-info-title">
+          <h4>Project Manager</h4>
+        </div>
+        <div className="col coa-details-info-right">
+          <CopyToClipboardHover
+            value={details.ppmProjectManager.email ?? ""}
+            id="ppmProjectManagerEmail"
+          >
+            {renderNameAndEmail(
+              details.ppmProjectManager.name,
+              details.ppmProjectManager.email
+            )}
+          </CopyToClipboardHover>
+        </div>
+      </div>
+      <div className="row">
+        <div className="col-3 coa-info-title">
+          <h4>Project Type</h4>
+        </div>
+        <div className="col coa-details-info-right">
+          <CopyToClipboardHover
+            value={details.projectTypeName}
+            id="projectTypeName"
+          >
+            {details.projectTypeName}
+          </CopyToClipboardHover>
+        </div>
+      </div>
```

Comment:
might not be worth it, up to you, but this exact section is just repeated with only a tiny difference.  Might be worth extracting into a component even if it isn't exported and only used within this file.  like `PpmDetailsRow` or something